### DISCUSSION TITLE
How to update?

### DISCUSSION THREAD

**Discussion by e-t-l** [ALTaleX531/TranslucentFlyouts#43] [Q&A] [answered]
To update this, e.g. from v2.0.0-alpha.3 to v2.0.0-alpha.4, does one need to first _uninstall_ the previous version, or can one simply downloaded the new version, unpack it, and run the .cmd file to overwrite the previous version?

**Comment by ALTaleX531** [comment, marked as answer]
Both ways are OK, but I recommend you to uninstall the previous version.
